packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -196,7 +196,6 @@ export default class ResizablePanelGroup extends Component<Signature> {
     nextPanelEl?: HTMLElement | null;
     prevPanelEl?: HTMLElement | null;
   } | null = null;
-  panelRatios: number[] = [];
```

Comment:
Removing this property could potentially bring back this [issue](https://linear.app/cardstack/issue/CS-6168/resizing-vertically-resizable-panel-container-causes-ratio-to-change). My purpose made this property is to cache the ratio length of all panels in a panel container/group before the container/browser window is being resized. When resizing the browser window, the size of the panels will be updated but with random ratio, then we will call `onContainerResize` to recalculate the size of the panels using the cache in the `panelRatio`. So if you use getter for the `panelRatios` and calculate the `panelRations` from current length of the panels, it might use the length of panels that is updated randomly because the resize event.

Reply:
Confirmed this is an issue that comes back

Reply:
Ahhh I know what you are trying to do. this.panelRatios is not eagerly recalculated

- panelRatios should be only calculated when the number of panels have changed (registering & unregistering)
- if we resize the window, the panelRatios should be cached

My change has an issue because I am recalculating after ANY container resize while number of panels are unchanging